src/Core/Services/SendService.cs

```diff
@@ -0,0 +1,270 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Bit.Core.Abstractions;
+using Bit.Core.Enums;
+using Bit.Core.Models.Data;
+using Bit.Core.Models.Domain;
+using Bit.Core.Models.Request;
+using Bit.Core.Models.Response;
+using Bit.Core.Models.View;
+using Newtonsoft.Json;
+
+namespace Bit.Core.Services
+{
+    public class SendService : ISendService
+    {
+        public static string SEND_KEY(string userId) => string.Format("sends_{0}", userId);
```

Comment:
Is this how we handle the key definitions in other services like cipher service or folder service?

Reply:
Not sure what you're referring to, but kind of. The keys look the same, the method of accessing it is slightly shorter.

This is how it's done in `FolderService`
```
private const string Keys_FoldersFormat = "folders_{0}";
```

accessed by
```
string.Format(Keys_FoldersFormat, userId)
```

I didn't like the `string.Format` all over the place so I just moved to a helper method

Reply:
I like the helper method, however let's match casing (so it doesn't look like a `const` named value) and name it as a function, `GetSendKey(string userId)` then I think this is good.